### PR TITLE
CDAP-15088 Add Aurora DB MySQL plugin

### PR DESCRIPTION
**Pull request by ibarbashov**

### CODE REVIEW

File: aurora-mysql-plugin/docs/AuroraMysql-action.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+**Port:** Port that MySQL master instance is listening to.
+
+**Database:** Aurora DB database name.
+
+**Username:** User identity for connecting to the specified database.
+
+**Password:** Password to use to connect to the specified database.
+
+**Connection Arguments:** A list of arbitrary string tag/value pairs as connection arguments. These arguments
```
Comment (jayfjin) [inline aurora-mysql-plugin/docs/AuroraMysql-action.md]: s/tag/key

File: aurora-mysql-plugin/docs/AuroraMysql-action.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+will be passed to the JDBC driver, as connection arguments, for JDBC drivers that may need additional configurations.
```
Comment (jayfjin) [inline aurora-mysql-plugin/docs/AuroraMysql-action.md]: remove the two commas in this line

File: aurora-mysql-plugin/docs/AuroraMysql-batchsink.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+**Database:** Aurora DB database name.
+
+**Table Name:** Name of the table to export to.
+
+**Username:** User identity for connecting to the specified database.
+
+**Password:** Password to use to connect to the specified database.
+
+**Connection Arguments:** A list of arbitrary string tag/value pairs as connection arguments. These arguments
```
Comment (jayfjin) [inline aurora-mysql-plugin/docs/AuroraMysql-batchsink.md]: s/tag/key

File: aurora-mysql-plugin/docs/AuroraMysql-batchsink.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+will be passed to the JDBC driver, as connection arguments, for JDBC drivers that may need additional configurations.
```
Comment (jayfjin) [inline aurora-mysql-plugin/docs/AuroraMysql-batchsink.md]: remove commas in this line

File: aurora-mysql-plugin/docs/AuroraMysql-postaction.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+**Connection Arguments:** A list of arbitrary string tag/value pairs as connection arguments. These arguments
+will be passed to the JDBC driver, as connection arguments, for JDBC drivers that may need additional configurations.
+
+**Auto Reconnect:** Should the driver try to re-establish stale and/or dead connections.
+
+Example
+-------
+Suppose you want to delete all records from an Aurora DB MySQL table "userEvents" of database "prod" running on 
+"mycluster.xyz.eu-central-1.rds.amazonaws.com", port 3306, without authentication using driver "mysql" if 
+pipeline completes successfully. Ensure that the driver for MySQL is installed (you can also provide driver name for
```
Comment (jayfjin) [inline aurora-mysql-plugin/docs/AuroraMysql-postaction.md:48]: s/if pipeline/if the pipeline

File: aurora-mysql-plugin/docs/AuroraMysql-action.md (hunk 1 above)
Comment (loganstk) [reply aurora-mysql-plugin/docs/AuroraMysql-action.md]: updated

File: aurora-mysql-plugin/docs/AuroraMysql-action.md (hunk 2 above)
Comment (loganstk) [reply aurora-mysql-plugin/docs/AuroraMysql-action.md]: updated

File: aurora-mysql-plugin/docs/AuroraMysql-batchsink.md (hunk 3 above)
Comment (loganstk) [reply aurora-mysql-plugin/docs/AuroraMysql-batchsink.md]: updated

File: aurora-mysql-plugin/docs/AuroraMysql-batchsink.md (hunk 4 above)
Comment (loganstk) [reply aurora-mysql-plugin/docs/AuroraMysql-batchsink.md]: updated

File: aurora-mysql-plugin/docs/AuroraMysql-postaction.md (hunk 5 above)
Comment (loganstk) [reply aurora-mysql-plugin/docs/AuroraMysql-postaction.md]: updated

File: aurora-mysql-plugin/docs/AuroraMysql-action.md (hunk 1 above)
Comment (ibarbashov) [reply aurora-mysql-plugin/docs/AuroraMysql-action.md]: updated

File: aurora-mysql-plugin/docs/AuroraMysql-action.md (hunk 2 above)
Comment (ibarbashov) [reply aurora-mysql-plugin/docs/AuroraMysql-action.md]: updated

File: aurora-mysql-plugin/docs/AuroraMysql-batchsink.md (hunk 3 above)
Comment (ibarbashov) [reply aurora-mysql-plugin/docs/AuroraMysql-batchsink.md]: updated

File: aurora-mysql-plugin/docs/AuroraMysql-batchsink.md (hunk 4 above)
Comment (ibarbashov) [reply aurora-mysql-plugin/docs/AuroraMysql-batchsink.md]: updated

File: aurora-mysql-plugin/docs/AuroraMysql-postaction.md (hunk 5 above)
Comment (ibarbashov) [reply aurora-mysql-plugin/docs/AuroraMysql-postaction.md:48]: updated

**Comment by ibarbashov** [comment]
@jayfjin should I also make the same changes in documentation for other plugins?

**Comment by jayfjin** [comment]
Yes, that would be awesome, thanks.

**Pull request merged**